doc/dev/developer_guide/testing_integration_tests/tests-integration-testing-teuthology-debugging-tips.rst

```diff
@@ -3,71 +3,83 @@
-  
-  nano /a/teuthology-2021-01-06_07:01:02-rados-master-distro-basic-smithi/
+and access `teuthology archives`_, like this for example:
 
-.. note:: This would require Sepia lab access. To know how to request it, see:
+  .. prompt:: bash $
+
+     nano /a/teuthology-2021-01-06_07:01:02-rados-master-distro-basic-smithi/
+
+.. note:: This requires you to have access to the Sepia lab. To learn how to
+          request access to the Speia lab, see: 
           https://ceph.github.io/sepia/adding_users/
 
-On pulpito, the jobs in red specify either a failed or dead job.
-Here, a job is combination of daemons and configurations that are formed using
+On pulpito, the jobs in red specify either a failed job or a dead job.
+A job is combination of daemons and configurations that are formed using
 `qa/suites`_ yaml fragments.
-Taking these configurations, teuthology runs the tasks that are present in
+Teuthology uses these configurations and runs the tasks that are present in
 `qa/tasks`_, which are commands used for setting up the test environment and
 testing Ceph's components.
-These tasks help us in covering large subset of usecase scenarios and hence
-exposing the bugs which were uncaught by `make check`_ testing.
+These tasks cover a large subset of usecase scenarios and help to  
+expose the bugs that were not caught by `make check`_ testing.
 
 .. _make check: ../tests-integration-testing-teuthology-intro/#make-check
 
-A job failure hence might be because of:
+A job failure might be caused by one or more of the following:
 
 * environment setup(`testing on varied systems<https://github.com/ceph/ceph/tree/master/qa/distros/supported>_`):
   testing compatibility with stable realeases for supported versions.
 
 * permutation of config values: for instance, qa/suites/rados/thrash ensures to
-  test Ceph under stressful workload, so that we be able to catch corner case
+  test Ceph under stressful workload, so that we are able to catch corner-case
   bugs.
-  The final setup config yaml that would be used for testing can be accessed
-  at::
+  The final setup config yaml used for testing can be accessed at::
 
   /a/<job-name>/<job-id>/orig.config.yaml
 
-More details about config.yaml can be found on `detailed test config`_
+More details about config.yaml can be found at `detailed test config`_
 
 Triaging the cause of failure
 ------------------------------
 
-To triage a job failure, open the teuthology log for it using either(from the
-pulpito page):
+To triage a job failure, open the teuthology log for it using either the job
+name or the job id (from the pulpito page):
+
+   http://qa-proxy.ceph.com/<job-name>/teuthology.log
+
+or
 
-http://qa-proxy.ceph.com/<job-name>/<job-id>/teuthology.log
+   http://qa-proxy.ceph.com/<job-id>/teuthology.log
 
-and then opening log file with signature as:
+Open the log file:
 
    /a/<job-name>/<job-id>/teuthology.log
 
+or
+
+   /a/<job-id>/teuthology.log
```

Comment:
I am not sure if this works.

Reply:
Thanks, Kefu. I'll see if I can't have it checked.

Reply:
@neha-ojha -- Can you check this part of the doc and confirm or deny that the teuthology logs for job names and job ids can be found at these locations?

I need this PR to be merged so that I can push four other PRs that also touch these files, and this PR is currently functionally holding the lock.

Reply:
```console
kchai@teuthology:~$ less /a/kchai-2021-01-30_16:51:13-rados-wip-kefu-testing-2021-01-30-2256-distro-basic-smithi/5842461/teuthology.log ^C
kchai@teuthology:~$ ls  /a/kchai-2021-01-30_16:51:13-rados-wip-kefu-testing-2021-01-30-2256-distro-basic-smithi/5842461/
config.yaml  console_logs  info.yaml  orig.config.yaml  owner  pid  teuthology.log  timing.yaml  worker.log
kchai@teuthology:~$ ls /a/5842461
ls: cannot access '/a/5842461': No such file or directory
```
i don't think we have automount-like per-user setting allowing each user has different result regarding to what they have under `/a`.

Reply:
I agree with @tchaikov, let's get rid of it.

Reply:
@neha-ojha Roger wilco.

Reply:
This was my fault; I misread the file and broke one working command into two non-working commands.

I put it back the way it was.